xeth/xeth.go

```diff
@@ -1044,6 +1044,7 @@ func (self *XEth) Transact(fromStr, toStr, nonceStr, valueStr, gasStr, gasPriceS
 	if err != nil {
 		return "", err
 	}
+	self.EthereumService().TxPool().SetLocal(signed)
```

Comment:
@zsfelfoldi, could you rebase and add this line in the new RPC method since XEth will be removed?
Location is `eth/api.go`.

Should this line also be added to the sendRawTransaction?